src/components/MovieList.vue

```diff
@@ -62,13 +56,13 @@ defineProps<{ movies: Movie[] }>();
 }
 
 .card {
-  max-width: 300px;
+  width: 300px;
   padding: 5px;
 }
 
 .card__img {
-  min-width: 300px; 
-  min-height: 450px;
+  width: 300px; 
+  height: 450px;
```

Comment:
Some elements don't have image, here is an example:

![image](https://github.com/AB619/vue-js-vite-lp/assets/16819882/0a08f472-1107-499b-8229-c06e2a197a0f)